### ISSUE TITLE
After closing, cannot open again without forcing quit

### ISSUE THREAD

**Issue by TylerLaBree** [gamingdoom/datcord#61]
I'm on the 0.5.1 flatpak, and when I close the app using the x in the top-right, Datcord remains running in the background. When I try to open it again using the app icon in gnome, or by using `flatpak run io.github.gamingdoom.Datcord`, I get the following popup window,

![image](https://github.com/gamingdoom/datcord/assets/51167014/7d7a2c59-3300-45e0-a3b4-bf80d5fddb9e)

and in the latter case, the following error,

```txt
Running in background
JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.
JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.
JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.
JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.
JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.
console.error: (new UnknownError("IndexedDB: main/url-classifier-skip-urls getLastModified() IndexedDB:   The operation failed for reasons unrelated to the database itself and not covered by any other error code.", "resource://services-settings/IDBHelpers.jsm", 18))
*** You are running in headless mode.
```

I must then close the app using gnome's background apps feature or `killall datcord` before I can open it again.

![image](https://github.com/gamingdoom/datcord/assets/51167014/c584559a-1a8c-472f-bb90-6fce8af53d2a)

**Comment by TylerLaBree** [comment]
This issue is referenced in https://github.com/gamingdoom/datcord/issues/46, so this is sort of a duplicate. Sorry!

**Comment by gamingdoom** [comment]
This issue will be fixed with 0.5.2 since that will allow reopening Datcord from the tray.

**Comment by Ambient-Impact** [comment]
Ah, that explains why I can't re-open without killing it first. Great work on this by the way, and as a web developer of 20 years and counting, I'm in total agreement that the last thing we want is Chrome to kill off Firefox/Gecko; I lived through the IE 6 days if you know about those dark times.

**Comment by gamingdoom** [comment]
Fixed with 0.5.2. You can now reopen it from the tray.
